docs/immortal-empires/tomb-kings.md

```diff
@@ -0,0 +1,34 @@
+# Tomb Kings
+
+## Arkhan the Black: Followers of Nagash
+
+* **Route I - Thy Kingdom Come**
+  * Arkhan has feigned obedience to Settra for long enough. The time has come to march out and retake the ancient lands of his master, the Great Necromancer Nagash.
+  * Occupy at least 6 provinces:
```

Comment:
We can not do AT LEAST X PROVINCES

Reply:
Change to occupy the following provinces: